test/functional/discovery_rules_controller_test.rb

```diff
@@ -3,6 +3,12 @@
 class DiscoveryRulesControllerTest < ActionController::TestCase
   setup :initialize_host
 
+  test "should add a link to navigation" do
+    get :index, params: {}, session: set_session_user
+    assert_response :success
+    assert_includes(response.body, '/discovery_rules/')
+  end
```

Comment:
You can drop this since https://github.com/theforeman/foreman_discovery/pull/618 removed the assertion and created a better one.